README.md

```diff
@@ -136,6 +136,13 @@ yarn install --frozen-lockfile
 yarn start
 ```
 
+In order to simulate WI-FI devices on your local network you can start application with 
+`MULTICAST_DNS_SIMULATOR_ENABLED` environment variable:
+
+```
+MULTICAST_DNS_SIMULATOR_ENABLED=true yarn start
```

Comment:
Use cross-env so the command will work across all operating systems:

npx cross-env MULTICAST_DNS_SIMULATOR_ENABLED=true yarn start